aiohttp/web_ws.py

```diff
@@ -615,6 +620,10 @@ async def receive_str(self, *, timeout: Optional[float] = None) -> str:
     async def receive_bytes(self, *, timeout: Optional[float] = None) -> bytes:
         msg = await self.receive(timeout)
         if msg.type is not WSMsgType.BINARY:
+            if msg.type not in [WSMsgType.CONTINUATION, WSMsgType.TEXT]:
+                raise WSMessageTypeError(
+                    f"Received message {msg.type}:{msg.data!r} has no content"
+                )
             raise TypeError(f"Received message {msg.type}:{msg.data!r} is not bytes")
```

Comment:
Maybe it makes more sense to not add the additional check and change the existing exception?

```suggestion
            raise WSMessageTypeError(f"Received message {msg.type}:{msg.data!r} is not bytes")
```

Reply:
Now that you have pointed this out, this does make more sense.

Reply:
Updated